core/src/main/java/org/apache/accumulo/core/rpc/clients/TServerClient.java

```diff
@@ -113,6 +129,9 @@ default Pair<String,C> getThriftServerConnection(Logger LOG, ThriftClientTypes<C
             TTransport transport = context.getTransportPool().getTransport(type,
                 tserverClientAddress, rpcTimeout, context, preferCachedConnections);
             C client = ThriftUtil.createClient(type, transport);
+            if (type == ThriftClientTypes.CLIENT && preferredClientHost != null) {
```

Comment:
If the preferred host does not exists, should it fall back to a random server?  If we don't want fallback could change the prop name to required host.

Reply:
I don't think it should fall back, as the property is explicitly set. This is why I added logging when the connection is successful. I'm going to add logging for the failures too.

Reply:
Added more logging in ae80003.

Reply:
I agree that the explicitly set value shouldn't fall back, so I agree it should have a name that indicates that it will be used, not that it's use is merely "preferred".

Reply:
@ctubbsii - are you suggesting that we should change the property name from `org.apache.accumulo.client.rpc.preferred.host` to something like `org.apache.accumulo.client.rpc.debug.host`?

Reply:
I'm going to make this change, I think it aligns better with the stated purpose.

Reply:
I like the `org.apache.accumulo.client.rpc.debug.host` name.

Reply:
Yes, that's definitely a better name. I like it.